src/components/seo.jsx

```diff
@@ -62,6 +66,10 @@ function SEO({
           name: 'twitter:description',
           content: metaDescription,
         },
+        {
+          property: 'twitter:image',
```

Comment:
Found [this example ](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image)from twitter, apparently they use `name` instead of `property` for it.
No idea why and when to use which though tbh.

```suggestion
          name: 'twitter:image',
```

Reply:
whoops, my bad. I'll have it fixed by tonight.